nixos/modules/security/pam.nix

```
@@ -383,6 +383,24 @@ let
         '';
       };
 
+      failDelay = {
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = lib.mdDoc ''
+            If enabled, this will replace the <literal>FAIL_DELAY</literal> setting from <literal>login.defs</literal>.
```

```suggestion
            If enabled, this will replace the `FAIL_DELAY` setting from `login.defs`.
```
That should work now.